infrastructure/critical/locals.tf

```diff
@@ -7,6 +7,13 @@ locals {
   public_subnets_new  = local.catalogue_vpcs["catalogue_vpc_delta_public_subnets"]
   private_subnets_new = local.catalogue_vpcs["catalogue_vpc_delta_private_subnets"]
 
+  catalogue_pipeline_ec_vpce_domain      = "vpce.eu-west-1.aws.elastic-cloud.com"
+  catalogue_pipeline_ec_privatelink_host = "${ec_deployment.pipeline_storage.id}.${local.catalogue_pipeline_ec_vpce_domain}"
```

Comment:
I can't see where this is being used?

Reply:
It's an [output](https://github.com/wellcomecollection/catalogue/pull/1261/files#diff-81d64fabe28d970892cf46ff6bc61d5b576d4e7cd2dfb6ceb16e7c5e985d29abR40).

Reply:
That it is - silly me.

Reply:
Services will use that address as the URI for their Elastic Cloud deployment, as it routes requests via the VPC endpoint (and the deployment ID tells Elastic Cloud the other side which deployment to reference).